spikingjelly/datasets/transform.py

```
@@ -34,6 +34,8 @@ def random_temporal_delete(
     :return: 长度为 `T_remain` 的序列，通过随机移除 `T - T_remain` 个切片获得
     :rtype: Union[torch.Tensor, np.ndarray]
 
+    :raises ValueError: 当 ``T_remain`` 为负数，或大于当前时间维长度时由 ``numpy.random.choice`` 抛出。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)